src/llama_stack_client/lib/cli/configure.py

```diff
@@ -1,94 +0,0 @@
-# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
Could we still keep configure? Or if you have a proposal for the alternative for using the CLI interface?
```
llama stack configure --endpoint 
```

Reply:
done.